silverback/_cli.py

```diff
@@ -15,17 +19,29 @@
 from silverback._click_ext import (
     SectionedHelpGroup,
     auth_required,
+    bot_path_callback,
     cls_import_callback,
     cluster_client,
     display_login_message,
     platform_client,
 )
-from silverback._importer import import_from_string
 from silverback.cluster.client import ClusterClient, PlatformClient
 from silverback.cluster.types import ClusterTier
 from silverback.runner import PollingRunner, WebsocketRunner
 from silverback.worker import run_worker
 
+DOCKERFILE_CONTENT = """
+FROM ghcr.io/apeworx/silverback:stable
+USER root
+WORKDIR /app
+RUN mkdir -p ./bots && chown harambe:harambe /app/bots
+USER harambe
+COPY ape-config.yaml .
+COPY requirements.txt .
+RUN pip install --upgrade pip && pip install -r requirements.txt
```

Comment:
We might need to configure this line based on whhich one of the 3 supported scenarios we detect

Reply:
`requirements.txt` :cry:

Reply:
ah yeah, it's missing `pyproject.toml`

we can just say the only place we look for packages is in `pyproject.toml`